lsp-tests/Test.hs

```diff
@@ -16,7 +17,8 @@ hoverTests = testGroup "Hover tests"
   [ testHover "mini.l4 hover LegalPractitioner"         "mini.l4" (Position 20 10) (mkRange 20 9 20 26) "TokenSym \"LegalPractitioner\""
   , testHover "mini.l4 hover the string 'lexicon'" "mini.l4" (Position 2 4)  (mkRange 2 0 2 7)    "This is a lexicon"
   , testHover "mini.l4 hover the string 'Business -> \"business_1_N\"'" "mini.l4" (Position 3 4)  (mkRange 3 0 3 26)    "This block maps variable Business to GrammaticalFramework WordNet definion \"business_1_N\""
-  , testHover "Hover over nothing" "mini.l4" (Position 1 0)  (mkRange 3 0 3 8)    "This block maps variable Business to WordNet definion \"business_1_N\""
+  , testHover "cr.l4 hover the string 'class Business {'" "cr.l4" (Position 21 10)  (mkRange 21 0 24 1)    "Declaration of new class : Business"
+  , expectFail $ testHover "Hover over nothing" "mini.l4" (Position 1 0)  (mkRange 3 0 3 8)    "This block maps variable Business to WordNet definion \"business_1_N\""
```

Comment:
We could add a reason here as well. Something like `expectFailBecause "Not implemented yet"` or something better if we can come up with something.